zerver/lib/notifications.py

```diff
@@ -26,6 +26,7 @@
 
 import datetime
 from email.utils import formataddr
+import lxml.html as LH
```

Comment:
I think we should just do `import lxml.html` or something; no reason to use an abbreviation here.